djangocms_navigation/views.py

```diff
@@ -0,0 +1,91 @@
+from django.apps import apps
+from django.contrib.contenttypes.models import ContentType
+from django.contrib.sites.shortcuts import get_current_site
+from django.core.exceptions import PermissionDenied
+from django.http import JsonResponse
+from django.views.generic import ListView
+
+from cms.models import Page
+
+from djangocms_navigation.utils import is_model_supported, supported_models
+
+
+class ContentObjectSelect2View(ListView):
```

Comment:
We expecting multiple results also we need to filter queryset based on a user query provided in UI which makes more sense to use `ListView` rather just `View`. As `ListView` has `get_queryset` method to override which I don't think `View` has.

Reply:
Yeah, but then you're overriding `get()` without using `super` and overriding `get_queryset` without `super` and not even setting any of the attributes that a `ListView` would expect (cause you've overridden everything that would error out without those attrs), so really you may as well make this officially a custom view by inheriting from `View` I think, no? Otherwise if somebody looks just at the class definition of this view (rather than the overridden methods), they might have an incorrect assessment of what this view is.